src/styles/flavors/material/dark.css

```
@@ -13,9 +13,43 @@
    --background_elevation_1A_static_idle-border-style: solid;
    --background_elevation_1A_static_idle-border-width: var(--size-size1);
 
+   --background_elevation_1B_static_idle-background-color: var(--background_B__03);
+   --background_elevation_1B_static_idle-box-shadow: 0 1px 1px 0 var(--grey_dark_1x),
+                                                     0 2px 1px -1px var(--grey_dark_1x),
+                                                     0 1px 3px 0 var(--grey_dark_1x);
+   --background_elevation_1B_static_idle-border-radius: var(--size-size4);
+
+   --background_elevation_3A_static_idle-background-color: var(--background_B__02);
+   --background_elevation_3A_static_idle-box-shadow: 0 0px 12px 0 var(--grey_dark_2x);
+
    --foreground_principal_no_background_static_idle-color: var(--foreground_B__01);
+
+  --table-color: var(--grey_light_3x);
+  --table-border-color: var(--grey_dark_3x);
+  --table-border-width: var(--size-size1);
+  --table-padding: 0 20px;
+
+  --input-border-width: var(--size-size2);
+  --input-border-color: var(--grey_light_3x);
+  --input-padding: var(--size-size3);
+
+  --navigation-link-color: var(--neutral_white);
+  --navigation-link-hover: rgba(255, 255, 255, 0.2);
  }
 
  @import '../generator.css';
 
  /* Specific theme styles go HERE */
+
+ .material-dark td, .material-dark th {
```

Please fix indentation
```suggestion
.material-dark td, .material-dark th {
```